### ISSUE TITLE
Repository name change

### ISSUE THREAD

**Issue by DulangaDasanayake** [DulangaDasanayake/dulangadasanayake.github.io#3]
Hello @codewithtyler what's up. I have changed the name of my repo. Since i'm not a pro on github like you you'll give me me some suggestions for change the name once again or revert it to first one.

I Have another request:

Can i join for a project that you're doing currently or in future?

- [ ] No
- [x] Yes

**Comment by codewithtyler** [comment]
I think portfolio is fine for the moment. Sorry I haven't gotten back to this project I started on it then got derailed into other projects that have taken up most of my time.

Yes, though I should share my current projects all involve AI in some way. Whether it's building apps fully with AI or AI assisting me. My current focus is mostly seeing what AI can do so that I have a better understanding how it can assist us in our careers. I don't think it's going to fully replace devs but if we don't know what it can and can't do we're gonna end up in a situation where we're falling behind in the industry unable to keep up.

**Comment by DulangaDasanayake** [comment]
_Good to hear that_. don't be sorry it's not a fault. I am so happy you agreed for collaborating on your projects. since i'm a kid to coding, i think you'll help me out on that.

Also i am using AI for coding. I think you already know that github Copilot is now provides a free version for developers. It's so good to have a senior developer in github.

Actually you are right. i think AI will definitely be a threat to developers in future in some way. Not only for that reason i think i have to rely on some other side like electronics. Computer hardware maybe.

I'm sorry if i didn't understand you properly. I have a bit mid level English knowledge it's a bit hard to learn a foreign like a country "Sri Lanka" that use Sinhala as main language you know.

_Once again thanks you so much !!_

**Comment by codewithtyler** [comment]
Yeah I've used Copilot before as well. If you got a small budget of say $40 per month it's worth looking into [Cursor](https://cursor.com) and [Bolt](https://bolt.new) as both of these are AI tools that can help speed up development.

Cursor forked VS Code and added LLM's on top of it to help you in your coding. It's more than what Copilot can do in the sense that Copilot can just make suggestions where as if you're not sure how to do something and need some help you can give Cursor's Composer feature the ability to view your code and tell it what you're trying to do and it can go help you write the code saving you a bunch of time.

Bolt on the other hand is web based tool that uses WebContainers to let you have a NodeJS developer environment in the browser. It's basically a way to let AI do the work for you. What I've started doing is taking my backlog of old ideas I've had and building them with Bolt and then when I get to a point I can't do something with Bolt I download the code pull into Cursor to debug manually and then use Cursor's AI when needed. It's changed how I think about coding because some days you just feel like you can't think straight about the problem and you need a bit of help well with side projects it's hard to find someone that can help you hence where AI comes into the mix.

**Comment by DulangaDasanayake** [comment]
Thanks for the detailed suggestions, Tyler! I’ll definitely look into Cursor and Bolt—they sound like game changers, especially for someone like me trying to streamline coding and learn more efficiently. The way you’re combining these tools to tackle projects is inspiring, and it’s great to hear how they’ve shifted your approach to development.

I completely agree—AI is a great assistant, especially when you’re stuck or working solo on side projects. I’ll try experimenting with these tools and see how they can fit into my workflow.
